pysolarmanv5/pysolarmanv5_async.py

```diff
@@ -1,11 +1,20 @@
 """pysolarmanv5_async.py"""
+
 import asyncio
 from multiprocessing import Event
-from typing import Any
 from umodbus.client.serial import rtu
 from .pysolarmanv5 import NoSocketAvailableError, PySolarmanV5
 
+# Disable `invalid-overridden-method` rule. The class `PySolarmanV5Async` overrides
+# (=redefines) non-async methods from `PySolarmanV5`.
+# The override in this case is not a problem in practice.
+
+# This could be avoided by changing the class hierarchy.
+# One way would be to introduce a common base class for async and sync classes
+# that would capture attributes and common (internal) sync methods.
```

Comment:
I'm open to discussing this in a future PR. :slightly_smiling_face:

Reply:
Let's see. Next I need to look into my battery management integration ...